scripts/release/create_changelog.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -E
+set -o pipefail
+
+CURRENT_RELEASE_TAG=$1
+LAST_RELEASE_TAG=$2
+DOCKER_IMAGE_URL=$3
+
+GITHUB_URL=https://api.github.com/repos/${CODE_REPOSITORY}
+GITHUB_AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
+CHANGELOG_FILE="CHANGELOG.md"
+
+echo "## What has changed" >> ${CHANGELOG_FILE}
+
+git log ${LAST_RELEASE_TAG}..${CURRENT_RELEASE_TAG} --pretty=tformat:"%h" --reverse | while read -r commit
+do
+    COMMIT_AUTHOR=$(curl -H "${GITHUB_AUTH_HEADER}" -sS "${GITHUB_URL}/commits/${commit}" | jq -r '.author.login')
+    git show -s ${commit} --format="* %s by @${COMMIT_AUTHOR}" >> ${CHANGELOG_FILE}
+done
```

Comment:
And another thing, have you consider using the automatically generated release notes via [`"generate_release_notes boolean": true`](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release)?

> Automatically generated release notes include a list of merged pull requests, a list of contributors to the release, and a link to a full changelog
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

Reply:
As for the 1) then yes, we'll have squashed commits, so only the PRs should be visible
As for the 2) then I haven't yet investigated on that. Sounds interesting though. Maybe we can create a follow-up issue? At this point it may be hard to compare how the changelog looks like, as we have no previous releases.